### DISCUSSION TITLE
(musescore) VERIFICATION.txt

### DISCUSSION THREAD

**Discussion by umol-unibas** [chocolatey-community/chocolatey-packages#2325] [General]
Hi there,

i'm not sure where to post this but i'll go here first. Would it be possible to "correct" the "VERIFICATION.txt" in the (musescore)-package to match the standards? In the txt-file are two msi-links included. As we have some automations running that check these links, it would be great if the first link could be rewritten from:

https://musescore.org/en/download/musescore.msi --> https://musescore.org/en/download/

As the link only leads to the main download-page for (musescore) this should not be of any issues. The download-link for the installer (second link in the txt-file) is okay.

Thanks!

**Comment by pauby** [comment]
There are no standard for VERIFICATION.txt files. I would suggest that the link that is there just now, https://musescore.org/en/download/musescore.msi, allows users to directly click on it and download the exact file that is being checksummed. Having the link be https://musescore.org/en/download/ means that people then need to search for the file.

**Comment by umol-unibas** [comment]
Thanks for your quick reply. Just to be clear: the correct link is available in the VERIFICATION.txt --> it is the second one. The users could click it ;) There's maybe no need for the first one, is it!? As the link ("https://musescore.org/en/download/musescore.msi ") in the update.ps1 is hard-coded it could be changed there, too ;) See [update.ps1](https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/musescore/update.ps1). Is it possible to change that? Thanks

**Reply by pauby** [reply]
Okay. I understand. 

I'd agree that the first one isn't necessary and the way it is worded suggests that the download location is sufficient.

My suggestion would be to raise an issue and a PR to fix it.

**Reply by umol-unibas** [reply]
Perfect - thank you! I'll create an issue and PR soon

**Reply by umol-unibas** [reply]
[issue](https://github.com/chocolatey-community/chocolatey-packages/issues/2327)
[PR](https://github.com/chocolatey-community/chocolatey-packages/pull/2326)

Done
